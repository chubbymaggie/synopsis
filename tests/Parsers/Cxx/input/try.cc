void func()
{
  try
  {
    cout << bar;
  }
  catch (string foo)
  {
    cout << "Error: " << foo << endl;
  }
  catch (...)
  {
    cout << "Catchall";
  }
}
