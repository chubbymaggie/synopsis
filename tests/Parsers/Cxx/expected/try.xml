<?xml version='1.0' encoding='ISO-8859-1'?>
<ast>
 <declarations>
  <instance accessibility="0" class="Synopsis.AST.Function" file="Parsers/Cxx/input/try.cc" language="C++" line="1" name="func()" realname="func" type="function">
   <returnType>
    <instance class="Synopsis.Type.Base" language="C++" name="void"/>
   </returnType>
  </instance>
 </declarations>
 <types>
  <instance class="Synopsis.Type.Declared" language="C++" name="func()">
   <declaration/>
  </instance>
 </types>
 <files>
  <file>
   Parsers/Cxx/input/try.cc
  </file>
 </files>
</ast>
